### ISSUE TITLE
Support project displayName

### ISSUE THREAD

**Issue by mfulton26** [palmerj3/jest-junit#39]
I'm running some Jest tests multiple times with different environment options. I have several files that look like the following but with different config file names, etc.:

```js
module.exports = {
  name: "Windows 10 Chrome 62.0 1024x768",
  displayName: "Windows 10 Chrome 62.0 1024x768",
  testEnvironment: "<rootDir>/WebDriverEnvironment.js",
  testEnvironmentOptions: {
    serverUrl: "http://hub-cloud.browserstack.com/wd/hub",
    capabilities: {
      "os": "Windows",
      "os_version": "10",
      "browserName": "Chrome",
      "browser_version": "62.0",
      "resolution": "1024x768",
      "browserstack.user": process.env.BROWSERSTACK_USER,
      "browserstack.key": process.env.BROWSERSTACK_ACCESSKEY,
      "browserstack.local": true,
    }
  }
};
```

They are each named something like `remote-{platform}-{browser}-{resolution}.config.js` so I have a final config file to run them all:

**jest.config.js**
```js
module.exports = {
  globalSetup: "<rootDir>/global.setup.js",
  globalTeardown: "<rootDir>/global.teardown.js",
  setupTestFrameworkScriptFile: "<rootDir>/test-framework.setup.js",
  projects: [
    "<rootDir>/remote-*.config.js"
  ],
  testResultsProcessor: "jest-junit"
};
```

The problem I am having though is that the junit.xml generated contains `<testsuite name="undefined"...` 😢 

Would is be possible to use the displayName for each project as the testSuiteName? If not, maybe the config file name/path? Thanks.

See also https://facebook.github.io/jest/docs/en/configuration.html#projects-array-string-projectconfig

**Comment by palmerj3** [comment]
@mfulton26 this is a really good point..

I will have to do some testing to confirm this - but the last time I tested jest multi-project with jest-junit I realized that jest does not give you any ability to pass information through to the testResultProcessor which would help to identify a suite..

That said I do think it's a really good idea to offer {displayName} as a template var which could be used for any of the fields.

If you feel like putting together a PR for this I would be happy to review it.
Otherwise I can take a look myself this weekend.

**Comment by palmerj3** [comment]
@mfulton26 I spent some time this morning and added the ability to use "displayName" in any of the template options. So you can now do something like this in your jest-junit config if you choose:

```
    "classNameTemplate": "{displayName}-{classname}-{title}",
    "titleTemplate": "{displayName}-{classname}-{title}",
```

This is available in version 3.6.0 which I just published to NPM.

**Comment by mfulton26** [comment]
Awesome! Thank you.
